cl/search/tasks.py

```diff
@@ -242,100 +244,33 @@ def person_first_time_indexing(parent_id: int, position: Position) -> None:
-    :return: SaveDocumentResponseType or None
+    :param instance_id: The primary key of the parent instance.
+    :param model: The model class of the instance.
+    :return: The object instance or None if it doesn't exist.
     """
-    es_args = {}
-    if isinstance(instance, Position):
-        parent_id = getattr(instance.person, "pk", None)
-        if not all(
-            [
-                es_index_exists(es_document._index._name),
-                parent_id,
-                # avoid indexing position records if the parent is not a judge
-                instance.person.is_judge,
-            ]
-        ):
-            return
-        if not PersonDocument.exists(id=parent_id):
-            person_first_time_indexing(parent_id, instance)
 
-        doc_id = ES_CHILD_ID(instance.pk).POSITION
-        es_args["_routing"] = parent_id
-    elif isinstance(instance, Person):
-        # index person records only if they were ever a judge.
-        if not instance.is_judge:
-            self.request.chain = None
-            return None
-        doc_id = instance.pk
-    elif isinstance(instance, RECAPDocument):
-        parent_id = getattr(instance.docket_entry.docket, "pk", None)
-        if not all(
-            [
-                es_index_exists(es_document._index._name),
-                parent_id,
-            ]
-        ):
-            self.request.chain = None
-            return None
-
-        if not DocketDocument.exists(id=parent_id):
-            # create the parent document if it does not exist in ES
-            docket_doc = DocketDocument()
-            doc = docket_doc.prepare(instance.docket_entry.docket)
-            DocketDocument(meta={"id": parent_id}, **doc).save(
-                skip_empty=False, return_doc_meta=True
-            )
-        doc_id = ES_CHILD_ID(instance.pk).RECAP
-        es_args["_routing"] = parent_id
-    else:
-        doc_id = instance.pk
-
-    es_args["meta"] = {"id": doc_id}
-    es_doc = es_document()
-    doc = es_doc.prepare(instance)
-    response = es_document(**es_args, **doc).save(
-        skip_empty=False,
-        return_doc_meta=True,
-        refresh=settings.ELASTICSEARCH_DSL_AUTO_REFRESH,
-    )
-    if type(instance) in models_alert_support and response["_version"] == 1:
-        # Only send search alerts when a new instance of a model that support
-        # Alerts is indexed in ES _version:1
-        if es_document == AudioDocument and not waffle.switch_is_active(
-            "oa-es-alerts-active"
-        ):
-            # Disable ES Alerts if oa-es-alerts-active switch is not enabled
-            self.request.chain = None
-            return None
-        return response["_id"], doc
-    else:
-        self.request.chain = None
+    try:
+        return model.objects.get(pk=instance_id)
+    except ObjectDoesNotExist:
+        logger.warning(
+            f"The {model.__name__} with ID {instance_id} doesn't exists and it"
+            f"can be updated in ES."
```

Comment:
Is this correct? Shouldn't it be:

```suggestion
            f"cannot be updated in ES."
```

Reply:
Yeah, this is correct, fixed.